include/Plot_GL.hpp

```diff
@@ -0,0 +1,32 @@
+#ifndef PLOT_HPP
+#define PLOT_HPP
+
+#include <atlas/glx/Buffer.hpp>
+#include <atlas/glx/Context.hpp>
+#include <atlas/glx/ErrorCallback.hpp>
+#include <atlas/glx/GLSL.hpp>
+#include <atlas/gui/GUI.hpp>
+#include <atlas/utils/Cameras.hpp>
+#include <atlas/utils/LoadObjFile.hpp>
+
+using namespace atlas;
```

Comment:
Careful with `using` declarations at global scope in your header files. Now this namespace is visible in every file that includes a file that includes this file!